contrib/netperf/collectd-td.spec.in

```diff
@@ -1833,8 +1836,16 @@ if [ $1 -eq 2 ]; then
 fi
 %systemd_post collectd.service
 %else
-/sbin/chkconfig --add cgconfig || :
-/sbin/chkconfig --add collectd-td || :
+/sbin/chkconfig --add cgconfig ||
+    echo 2>&1 '"chkconfig --add cgconfig" failed (keep going)'
+/sbin/chkconfig --add collectd-td ||
+    echo 2>&1 '"chkconfig --add collectd" failed (keep going)'
+levels=`/bin/sed -r -n 's/^ *# *chkconfig: *([0-9]+).*$/\1/p' \
+        /etc/init.d/collectd-td`
+/sbin/chkconfig --levels="$levels" cgconfig on ||
```

Comment:
On Sun, Sep 6, 2015 at 2:13 PM, yosih <notifications@github.com> wrote:

> In contrib/netperf/collectd-td.spec.in
> <https://github.com/GoogleCloudPlatform/collectd-netperf/pull/18#discussion_r38820242>
> :
>
> > @@ -1833,8 +1836,16 @@ if [ $1 -eq 2 ]; then
> >  fi
> >  %systemd_post collectd.service
> >  %else
> > -/sbin/chkconfig --add cgconfig || :
> > -/sbin/chkconfig --add collectd-td || :
> > +/sbin/chkconfig --add cgconfig ||
> > +    echo 2>&1 '"chkconfig --add cgconfig" failed (keep going)'
> > +/sbin/chkconfig --add collectd-td ||
> > +    echo 2>&1 '"chkconfig --add collectd" failed (keep going)'
> > +levels=`/bin/sed -r -n 's/^ *# *chkconfig: *([0-9]+).*$/\1/p' \
> > +        /etc/init.d/collectd-td`
> > +/sbin/chkconfig --levels="$levels" cgconfig on ||
>
> My man says "chkconfig [--level levels] [--type type] name "
> (yosih8-centos6). Does --levels="" work?
>

Welcome to the wonderful world of init scripts! Both work and do the same
thing. --level 2345 scares me, because it looks like a really high runlevel.

-- 
Ariel.